parlai/mturk/core/dev/agents.py

```diff
@@ -177,9 +104,31 @@ class MTurkAgent(Agent):
     def __init__(self, opt, mturk_manager, hit_id, assignment_id, worker_id):
         super().__init__(opt)
 
-        self.conversation_id = None
-        self.mturk_manager = mturk_manager
+        # all MTurkManager functions explicitly used by agents extracted here
+        self.m_send_state_change = mturk_manager.send_state_change
+        self.m_send_message = mturk_manager.send_message
+        self.m_handle_turker_timeout = mturk_manager.handle_turker_timeout
+        self.m_send_command = mturk_manager.send_command
+        # TODO update or remove this function
+        self.m_get_agent_work_status = mturk_manager.get_agent_work_status
```

Comment:
This function should be replaced when I update to a better assignment state tracking mechanism